packages/propel/src/combobox/combobox.stories.tsx

```diff
@@ -0,0 +1,253 @@
+import type { Meta, StoryObj } from "@storybook/react-vite";
+import { useState } from "react";
+import { Combobox } from "./combobox";
+import { Check, ChevronsUpDown } from "lucide-react";
+
+const meta: Meta = {
+  title: "Components/Combobox",
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+};
+
+export default meta;
+type Story = StoryObj;
+
+const frameworks = [
+  { value: "react", label: "React" },
+  { value: "vue", label: "Vue" },
+  { value: "angular", label: "Angular" },
+  { value: "svelte", label: "Svelte" },
+  { value: "solid", label: "Solid" },
+  { value: "next", label: "Next.js" },
+  { value: "nuxt", label: "Nuxt" },
+  { value: "remix", label: "Remix" },
+];
+
+export const Default: Story = {
+  render: function DefaultCombobox() {
+    const [value, setValue] = useState("");
+
+    return (
+      <Combobox value={value} onValueChange={(v) => setValue(v as string)}>
```

Comment:
@anmolsinghbhatia is it possible to have `string[]` in both multi and single select Combobox? if not we should discriminate this